src/nrc/api/tasks.py

```diff
@@ -66,9 +67,13 @@ def deliver_message(sub_id: int, msg: dict, **kwargs) -> None:
 
         # Only log if a top-level object is provided
         if notificatie_id:
-            NotificatieResponse.objects.create(
-                notificatie_id=notificatie_id,
-                abonnement=sub,
-                attempt=kwargs.get("attempt", 1),
-                **response_init_kwargs
-            )
+            try:
+                NotificatieResponse.objects.create(
+                    notificatie_id=notificatie_id,
+                    abonnement=sub,
+                    attempt=kwargs.get("attempt", 1),
+                    **response_init_kwargs
+                )
+            except DataError:
```

Comment:
Instead of skipping storing the NotificatieResponse upon this error, I think the way to go would be to store at most 1000 characters for the exception_message.

Reply:
It should be ok now